orange/src/main/java/com/hartwig/hmftools/orange/report/ReportWriterFactory.java

```diff
@@ -12,11 +12,11 @@ private ReportWriterFactory() {
     @NotNull
     public static ReportWriter createToDiskWriter(@NotNull OrangeConfig config) {
         String outputDir = config.outputDir();
-        return new ReportWriter(true, outputDir, new PlotPathResolver(outputDir));
+        return new ReportWriter(true, outputDir, new PlotPathResolver(outputDir), config.addDisclaimer());
     }
 
     @NotNull
     public static ReportWriter createInMemoryWriter() {
-        return new ReportWriter(false, null, new PlotPathResolver(null));
+        return new ReportWriter(false, null, new PlotPathResolver(null), false);
```

Comment:
Minor detail but would set this the disclaimer to `true`. the in-memory writer should have everything enabled if possible, leads to a better test (e.g. to capture a null-pointer in the section enabled by the disclaimer)